NotationV0/task.json

```diff
@@ -0,0 +1,198 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
+    "id": "f7d56ba6-35a6-453d-8192-8b602bf234bd",
+    "name": "notation",
+    "friendlyName": "Notation",
+    "description": "Azure Pipepine Task for setting up Notation CLI, sign and verify with Notation",
+    "helpMarkDown": "",
+    "category": "Utility",
+    "author": "Notation Developers",
+    "version": {
+        "Major": 0,
+        "Minor": 2,
+        "Patch": 75
+    },
```

Comment:
How are the versions defined?

Reply:
Each time when I test the code, I need to 
1. increase the version number
2. build and publish the task to a private Marketplace
3. run the task in the pipeline

We can create an issue to reset the version before we push to the azure-pipelines-tasks repo.

Reply:
#2